spring-cloud-commons/src/main/java/org/springframework/cloud/client/loadbalancer/reactive/RetryableLoadBalancerExchangeFilterFunction.java

```diff
@@ -66,22 +67,46 @@
 
 	private final ReactiveLoadBalancer.Factory<ServiceInstance> loadBalancerFactory;
 
+	private final List<LoadBalancerClientRequestTransformer> transformers;
+
+	/**
+	 * Deprecated in favor of
+	 * {@link #RetryableLoadBalancerExchangeFilterFunction(LoadBalancerRetryPolicy, ReactiveLoadBalancer.Factory, LoadBalancerRetryProperties, List)}.
+	 * @param retryPolicy the retry policy
+	 * @param loadBalancerFactory the loadbalancer factory
+	 * @param retryProperties the retry properties
+	 */
+	@Deprecated
 	public RetryableLoadBalancerExchangeFilterFunction(
 			LoadBalancerRetryPolicy retryPolicy,
 			ReactiveLoadBalancer.Factory<ServiceInstance> loadBalancerFactory,
 			LoadBalancerRetryProperties retryProperties) {
-		this.retryPolicy = retryPolicy;
-		this.loadBalancerFactory = loadBalancerFactory;
-		this.retryProperties = retryProperties;
+		this(retryPolicy, loadBalancerFactory, retryProperties, Collections.emptyList());
 	}
 
+	/**
+	 * Deprecated in favor of
```

Comment:
Let's use: `@deprecated` tag in javadoc.

Reply:
Done https://github.com/spring-cloud/spring-cloud-commons/pull/925/commits/e9a7ab92aaf07d567000374a7f1365c2b0dc4ba4